timemachine/fe/bar.py

```diff
@@ -196,24 +202,24 @@ def bootstrap_bar(
         )
         bootstrap_samples.append(bar_result)
 
-    return full_bar_result, np.array(bootstrap_samples)
+    return full_bar_result, full_bar_err, np.array(bootstrap_samples)
 
 
 def bar_with_bootstrapped_uncertainty(
```

Comment:
The name of this function now seems a bit misleading, since it doesn't always return a bootstrapped uncertainty. Should it just be `estimate_bar_and_uncertainty` or similar?